anghammarad/src/main/scala/com/gu/anghammarad/AnghammaradService.scala

```diff
@@ -6,7 +6,7 @@ import com.gu.anghammarad.messages.{Messages, SendMessages}
 import scala.util.Try
 
 
-object Anghammarad {
+object AnghammaradService {
```

Comment:
It was necessary to rename this as the client site inside the `package com.gu.anghammarad` making it impossible to use them when included together (as in the `dev` project).